packages/components/src/components/dialog/dialog.jsx

```diff
@@ -6,126 +6,122 @@ import { CSSTransition } from 'react-transition-group';
+                                <Icon icon='IcCross' />
+                            </div>
+                        )}
+                    </div>
+                    {typeof children === 'string' ? (
+                        <p className={content_classes}>{children}</p>
+                    ) : (
+                        <div className={content_classes}>{children}</div>
+                    )}
+                    <div className='dc-dialog__footer'>
+                        {!!onCancel && (
+                            <Button
+                                className='dc-dialog__button'
+                                has_effect
+                                text={cancel_button_text}
+                                onClick={handleCancel}
+                                secondary
+                                large
+                            />
+                        )}
+                        {!!confirm_button_text && (
+                            <Button
+                                className='dc-dialog__button'
+                                has_effect
+                                text={confirm_button_text}
+                                onClick={handleConfirm}
+                                primary
+                                large
+                            />
                         )}
-                        <div className='dc-dialog__footer'>
-                            {!!onCancel && (
-                                <Button
-                                    className='dc-dialog__button'
-                                    has_effect
-                                    text={cancel_button_text}
-                                    onClick={this.handleCancel}
-                                    secondary
-                                    large
-                                />
-                            )}
-                            {!!confirm_button_text && (
-                                <Button
-                                    className='dc-dialog__button'
-                                    has_effect
-                                    text={confirm_button_text}
-                                    onClick={this.handleConfirm}
-                                    primary
-                                    large
-                                />
-                            )}
-                        </div>
                     </div>
                 </div>
-            </CSSTransition>
-        );
-
-        if (portal_element_id) {
-            return ReactDOM.createPortal(dialog, document.getElementById(portal_element_id));
-        }
+            </div>
+        </CSSTransition>
+    );
 
-        return dialog;
+    if (portal_element_id) {
+        return ReactDOM.createPortal(dialog, document.getElementById(portal_element_id));
     }
-}
+
+    return dialog;
+};
```

Comment:
Some propsTypes are missing         
        is_mobile_full_width,
        is_content_centered,
        has_close_icon,

Reply:
Thank you, I added the missing propTypes